modules/appeals_api/spec/services/appeals_api/evidence_submission/file_validator_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe AppealsApi::EvidenceSubmission::FileValidator do
+  describe '#validate' do
+    let(:fixtures_path) { '/modules/appeals_api/spec/fixtures' }
+
+    let(:valid_pdf) { File.new("#{::Rails.root}#{fixtures_path}/expected_10182_extra.pdf") }
+    let(:non_pdf) { File.new("#{::Rails.root}#{fixtures_path}/test_incorrect_type.docx") }
+    let(:pretend_pdf) { File.new("#{::Rails.root}#{fixtures_path}/test_fake_pdf.pdf") }
+    let(:oversize_pdf) { File.new("#{::Rails.root}#{fixtures_path}/oversize_11x17.pdf") }
+
+    context 'when supported file uploaded (100MB 11in x 11in PDF)' do
+      it 'does not raise an error' do
```

Comment:
To me, the word "raise" is something specific around exceptions - e.g `expect(x).to raise_error XYZ`. This is returning structured data about an error, not raising.

Something like `returns error data when...` is clearer to me for these types of tests.

Reply:
Nice. Thank you!